common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -266,28 +273,21 @@ def test_email_field(self):
             assert_after_reload=False
         )
 
-        self.verify_events_of_type(
-            username,
-            self.CHANGE_INITIATED_EVENT_NAME,
+        actual_events = self.wait_for_events(
+            event_filter=self.settings_change_initiated_event_filter, number_of_matches=2)
```

Comment:
Yeah, I can't really think of a clean way to catch this type of error. Open to ideas...